src/test/java/ArrayQueueEqualsTest.java

```diff
@@ -47,23 +47,6 @@ public void equals_verify_contract() {
         gSingletonDefault.enqueue(100);
         gSingletonCapacity.enqueue(100);
 
-        new EqualsTester()
-                .addEqualityGroup(ArrayQueue.class)
-                .addEqualityGroup(gEmptyDefault,
-                        gEmptyCapacity)
-                .addEqualityGroup(gSingletonDefault,
-                        gSingletonCapacity)
-                .addEqualityGroup(gManyOneDefaultA,
-                        gManyOneDefaultB,
-                        gManyOneCapacity,
-                        gManyOneFront,
-                        gManyOneRear)
-                .addEqualityGroup(gManyTwoDefault,
-                        gManyTwoCapacity)
-                .addEqualityGroup(gReverseDefault,
-                        gReverseCapacity)
-                .addEqualityGroup(gTypeDefault,
-                        gTypeCapacity)
-                .testEquals();
+        new EqualsTester().addEqualityGroup(ArrayQueue.class).addEqualityGroup(gEmptyDefault, gEmptyCapacity).addEqualityGroup(gSingletonDefault, gSingletonCapacity).addEqualityGroup(gManyOneDefaultA, gManyOneDefaultB, gManyOneCapacity, gManyOneFront, gManyOneRear).addEqualityGroup(gManyTwoDefault, gManyTwoCapacity).addEqualityGroup(gReverseDefault, gReverseCapacity).addEqualityGroup(gTypeDefault, gTypeCapacity).testEquals();
```

Comment:
Probably want to set it to put new lines in big method chains like this. This is hard to read.

Reply:
:eyes: